squid-mixin/g.libsonnet

```
@@ -0,0 +1,3 @@
+// grafonnet must be imported with "g" alias
+local g = import './vendor/github.com/grafana/grafonnet/gen/grafonnet-v11.0.0/main.libsonnet';
+g
```

```suggestion
// grafonnet must be imported with "g" alias
local g = import './vendor/github.com/grafana/grafonnet/gen/grafonnet-v11.0.0/main.libsonnet';
g
```

This is the wrong import syntax: example https://github.com/grafana/jsonnet-libs/blob/master/apache-activemq-mixin/g.libsonnet